### DISCUSSION TITLE
Adding a second movie library

### DISCUSSION THREAD

**Discussion by j-ktz** [meisnate12/Plex-Meta-Manager#75] [Q&A]
I've modified my config to add a second movie library "Movies - Kids" and a second yml for "Movies - Kids.yml", yet it doesn't seem to scan it and add collections. Is this because of the cache?

**Comment by meisnate12** [comment]
post your log so i can see what's going on

**Reply by j-ktz** [reply]
Attached! 

[meta-13.log](https://github.com/meisnate12/Plex-Meta-Manager/files/6057910/meta-13.log)

**Reply by meisnate12** [reply]
the metadata file for the kids library seems to be having an error around line 8

**Reply by j-ktz** [reply]
Okay now it's saying "Collection Error: Studio attribute not supported". Do I have an error here?

Pixar:
    Studio: Pixar, Pixar Animation Studios, Pixar/Disney
    summary: Movies by Pixar
    collection_order: alpha
    sort_title: +++Pixar

**Reply by j-ktz** [reply]
I was missing plex-search. Got it now!

**Reply by meisnate12** [reply]
it would have probably worked with `studio` as well the program is case sensitive

**Reply by j-ktz** [reply]
Something I'm seeing with
"plex_search:
  studio: Pixar, Pixar Animation, Pixar Animation Studios"

Is that it'll also pickup "Disney Pixar", is there a way to put the studio names in quotes to prevent this? I ONLY want "Pixar" not "Disney Pixar" but it finds Pixar in as the second word and adds it to the Pixar Collection when I don't want it to.

**Reply by j-ktz** [reply]
Still having this issue. Any idea @meisnate12?

**Reply by meisnate12** [reply]
it just uses plex's default search which for studio will contain Disney Pixar. You could try adding an inverse filter for the studio you don't want like this

```yaml
studio: Pixar, Pixar Animation, Pixar Animation Studios
filters:
  studio.not: Disney Pixar
```

**Reply by j-ktz** [reply]
Thanks I'll try that! Is it case sensitive? For example, could I do disney pixar, or do I need it exactly as it is in Plex?

**Reply by meisnate12** [reply]
the filter needs to be exact and is case sensitive the plex search works however their search works I'm not sure tbh I just send plex the data and it does its thing
